src/plugins/dashboard_embeddable_container/public/embeddable/grid/_dashboard_grid.scss

```diff
@@ -34,7 +34,7 @@
  .dshLayout-isMaximizedPanel {
   height: 100% !important; /* 1. */
   width: 100%;
-  position: absolute;
+  position: absolute !important;
```

Comment:
Why is this needed?

Reply:
Same reason it's needed for the height - to override the react grid layout.